.github/workflows/presubmit.yaml

```diff
@@ -101,6 +101,29 @@ jobs:
       with:
         name: debs
         path: debs.tar
+  e2e-tests-orchestration-build-image:
+    runs-on: ubuntu-24.04
+    if: ${{ always() && needs.build-debian-package.result == 'success' }}
+    needs: [build-debian-package]
```

Comment:
nit: If possible this should be moved one line up. IIUC the `if:` statement is using the dependency declared here, it's more readable to declare the dependency before referencing it.